mne/channels/layout.py

```diff
@@ -370,7 +369,7 @@ def find_layout(info, ch_type=None, exclude='bads'):
                        FIFF.FIFFV_COIL_CTF_OFFDIAG_REF_GRAD)
     has_CTF_grad = (FIFF.FIFFV_COIL_CTF_GRAD in coil_types or
                     (FIFF.FIFFV_MEG_CH in channel_types and
-                     any([k in ctf_other_types for k in coil_types])))
+                     any(k in ctf_other_types for k in coil_types)))
     # hack due to MNE-C bug in IO of CTF
     n_kit_grads = len([ch for ch in chs
                        if ch['coil_type'] == FIFF.FIFFV_COIL_KIT_GRAD])
```

Comment:
If we're really shooting to use generators, could this just be `sum(ch['coil_type'] = FIFF.FIFFV_COIL_KIT_GRAD for ch in chs)`?